src/views/blocks/tickets/submit-button-modal.php

```diff
@@ -14,14 +14,17 @@
  * @since 4.11.3 Updated the button to include a type - helps avoid submitting forms unintentionally.
  * @since 4.11.3 Allow filtering of the button classes.
  * @since 4.11.3 Added button ID for better JS targeting.
+ * @since TBD Add support for custom label for "Tickets" plural.
  *
- * @version 4.11.3
+ * @version TBD
  *
  */
 
-/* translators: %s is the event or post title the tickets are attached to. */
-$title       = sprintf( _x( '%s Tickets', 'Modal title. %s: event name', 'event-tickets' ), get_the_title() );
-$button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
+/* translators: %1$s: Event name, %2$s: Tickets label */
+$title = sprintf( _x( '%1$s %2$s', 'Tickets modal title.', 'event-tickets' ), get_the_title(), tribe_get_ticket_label_plural( 'event-tickets' ) );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: b355865c7fe6d23fdca56801a2f8596e</sub>